client/.env.example

```
@@ -0,0 +1,11 @@
+# Firebase Configuration
+# Get these values from your Firebase project settings
+REACT_APP_FIREBASE_API_KEY=your_api_key_here
+REACT_APP_FIREBASE_AUTH_DOMAIN=your_project_id.firebaseapp.com
+REACT_APP_FIREBASE_PROJECT_ID=your_project_id
+REACT_APP_FIREBASE_STORAGE_BUCKET=your_project_id.appspot.com
+REACT_APP_FIREBASE_MESSAGING_SENDER_ID=your_sender_id
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The REACT_APP_FIREBASE_MESSAGING_SENDER_ID key should go before the REACT_APP_FIREBASE_PROJECT_ID key

(UnorderedKey)